packages/plugin-dom/Dom.ts

```diff
@@ -221,8 +222,12 @@ export class Dom extends JWPlugin {
 
         let node = this.editor.vDocument.root.lastLeaf();
         while (node) {
-            const renderedNode = (await this.editor.renderers.dom.render(node)) as Node[];
-            for (const domNode of renderedNode) {
+            const domEngine = this.editor.renderers.dom as RenderingEngine<Node[]>;
+            await domEngine.render(node);
+            const renderings = domEngine.renderings.get(node);
+            const rendering = renderings[renderings.length - 1];
+            const renderedNodes = await rendering[1];
```

Comment:
You don't have to `await domEngine.render(node)` if you already `await rendering[1]` anyway. Just call render to be sure that the rendering is triggered if it was not already, but you don't have to wait for it. We'll gain one microtask tick per node of the tree. Not much, I agree, but we tend to have a lot of nodes.